shared/components/initialScene.js

```diff
@@ -0,0 +1,184 @@
+import React, {Component, PropTypes} from 'react';
+import {Image, Platform, View, Text, Dimensions, TouchableOpacity} from 'react-native';
+import Swiper from 'react-native-swiper';
+import { Actions } from 'react-native-router-flux';
+
+import ImgLogin1 from '../resources/initialScene/loginImage1.png';
+import ImgLogin2 from '../resources/initialScene/loginImage2.png';
+import ImgLogin3 from '../resources/initialScene/loginImage3.png';
+
+const WindowHeight = 477.8 + 162;
+const WindowWidth = 360;
```

Comment:
so, these values won't directly applied to the component's styles as you will see below